cirq/testing/consistent_act_on.py

```diff
@@ -41,19 +44,32 @@ def state_vector_has_stabilizer(state_vector: np.ndarray,
     return np.allclose(args.target_tensor, state_vector)
 
 
-def assert_act_on_clifford_tableau_effect_matches_unitary(val: Any) -> None:
-    """Checks that act_on with CliffordTableau generates stabilizers that
-    stabilize the final state vector. Does not work with Operations or Gates
-    expecting non-qubit Qids."""
+def assert_all_implemented_act_on_effects_match_unitary(
+        val: Any,
+        assert_tableau_implemented: bool = False,
+        assert_ch_form_implemented: bool = False) -> None:
+    """
```

Comment:
Please fix the docs string according to our [style guide](https://github.com/quantumlib/Cirq/blob/master/docs/dev/development.md#writing-docstrings-and-generating-documentation).

Namely: no single line intro

Reply:
Done